File: TMessagesProj/src/main/java/org/ushastoe/fluffy/hooks/HookBridgeTemplate.java
package org.ushastoe.fluffy.hooks;

import org.ushastoe.fluffy.patches.FluffyFeaturePatch;

public final class HookBridgeTemplate {
    private HookBridgeTemplate() {
    }

    public static void applyFeaturePatch(Object target) {
        FluffyFeaturePatch.apply(target);
    }
}

File: TMessagesProj/src/main/java/org/ushastoe/fluffy/hooks/HookBridgeTemplate.kt
package org.ushastoe.fluffy.hooks

import org.ushastoe.fluffy.patches.FluffyFeaturePatch

object HookBridgeTemplate {
    @JvmStatic
    fun applyFeaturePatch(target: Any?) {
        FluffyFeaturePatch.apply(target)
    }
}
